techlibs/intel_alm/synth_intel_alm.cc

```diff
@@ -43,6 +43,7 @@ struct SynthIntelALMPass : public ScriptPass {
 		log("    -family <family>\n");
 		log("        target one of:\n");
 		log("        \"cyclonev\"    - Cyclone V (default)\n");
+		log("        \"arriav\"      - Arria V (non-GZ)");
```

Comment:
I wonder if it would be a good idea to add `arriavgx` as an alias (to avoid confusion why there eventually is `arriav` and `arriavgz`)

Reply:
I think that could be too specific. Remember all the other models of non-GZ Arria V; `arriavgx` could give the impression that only that specific family is supported. On the other hand, if someone with a GZ selects `arriav` instead of either `arriavgz` or (more likely) `stratixv`, the design will still be implementable on a GZ.

Reply:
... right, I confused it with some other family

what about adding aliases `arriavgx`, `arriavgt`, `arriavsx`, `arriavst`? there aren't that many of them

also, arria V GX design will not be implementable on arria V GZ due to GX using M10K and GZ using M20K...

Reply:
This would be too unwieldy to explain in the documentation, I think. If it can't be explained well enough, the feature might as well not exist.